source/ref_installation.md

```diff
@@ -0,0 +1,415 @@
+# ODAHU installation from scratch
+
```

Comment:
Let's specify Odahu prerequisites. I mean:
* kuberentes cluster
* object storage(s3, gcs, abs)

Reply:
Done in c35c683823bc4bedc40272cafa686519adcc5856.